.github/workflows/validate_all_yml.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+name: validate all yaml files
+on: [push]  # yamllint disable-line rule:truthy
```

Comment:
`# yamllint disable-line rule:truthy` adding this to all our file, will soon become a pain, when people will wonder why this is failing. We should either add a config file for this yamlint that allow this `rule:truthy` same for the line length limitation. Instead of having people remember to explicitely disable that route. Especially for the `rule:truthy` that does not seem an interesting rule to apply here.